backend/src/govsg/models/govsg-message-transactional.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  BelongsTo,
+  Column,
+  DataType,
+  ForeignKey,
+  Model,
+  Table,
+} from 'sequelize-typescript'
+import { GovsgTemplate } from './govsg-template'
+import { GovsgMessageStatus } from '@core/constants'
+
+@Table({
+  tableName: 'govsg_messages_transactional',
+  underscored: true,
+  timestamps: true,
+})
+export class GovsgMessageTransactional extends Model<GovsgMessageTransactional> {
+  @ForeignKey(() => GovsgTemplate)
+  @Column(DataType.BIGINT)
+  templateId: number
+
+  @BelongsTo(() => GovsgTemplate)
+  template: GovsgTemplate
```

Comment:
now that I am looking at the WhatsApp API more closely, there are two kinds of messages:
- template messages (which makes sense to have this relationship with GovsgTemplate
- text messages (which are custom text messages that you can send)

We are only supporting the first type for our MVP, but I wonder if:
- we should rename this table to indicate that it only covers the first type of message (e.g. `GovsgTemplateMessageTransactional`)
- or we should redesign or data model here to support both kinds (or would that be premature optimisation)

Reply:
Our `GovsgMessage` is already the first type and similar concept across the code base also refers to the same thing, hence adding `Template` would be redundant

Reply:
no what I mean is, in the future, if we send transactional `text messages` (which is fully custom messages that don't need to follow any templates), then it might be a bit confusing what to name that table hmm

Reply:
(this is in contrast to other channels like SMSes and emails, where the content of the transactional messages are alr fully custom)